lib/freno/client/requests/check.rb

```diff
@@ -16,6 +16,12 @@ def initialize(**kwargs)
             present app: app, store_type: store_type, store_name: store_name
           end
 
+          # A low priority check is handled slightly differently by Freno. If
+          # the p=low GET parameter is passed, the check will fail for any app
+          # with failed checks within the last second. This failure is returned
+          # quickly, without checking the underlying metric.
+          params[:p] = "low" if options[:low_priority]
```

Comment:
Should we add docs for the `low_priority` option at both the `check`/`check_read` and `client.options` level?

Reply:
Done!